### ISSUE TITLE
Camera delayed?

### ISSUE THREAD

**Issue by b1naryv1k1ng** [mrlt8/docker-wyze-bridge#475]
For some reason, the feed from the camera is EXTREMELY delayed. I moved a Wyze Outdoor cam inside almost minutes later, its still showing the outside. When I look in the wyze app itself, its showing inside. 

Is there anything I can change to remove this delay? Are there any logs in the docker container that might shed more light on whats going on?

**Comment by mrlt8** [comment]
Is this with the RTSP stream? Some players cache the video to prevent buffering which could result in a delayed video.

Are you rotating the video in the bridge by any chance? There were some issues with the host machine not having enough power to process the video #452

**Comment by b1naryv1k1ng** [comment]
Yes, its the RTSP stream. Its not rotated at all. I've got the RTSP feed going into Blue Iris and I noticed the stream was over 10 minutes delayed.

**Comment by RyanHope** [comment]
My doorbell cam is 10 minutes delayed.

**Comment by tmeekes** [comment]
And here I was thinking a 10 second delay in the Wyze-Bridge UI RTSP stream was something I needed to dig in to...

FYI - I get a discrepancy between the Wyze-Bridge UI and my Agent DVR UI that is reading the stream... Agent has a 4 second delay and Wyze-Bridge has a 10 second delay. I'm putting it on player video buffering like @mrlt8 says... would explain the discrepancies in UIs

**Comment by jappish84** [comment]
I've used wyze-Bridge for years without issues but also seeing this issue now where stream is delayed by about 5 minutes. Using wyze cam v2 on latest fw
